src/mem/cache/cache_blk.hh

```diff
@@ -466,6 +474,10 @@ class CacheBlk : public TaggedEntry
     /** Set the source requestor id. */
     void setSrcRequestorId(const uint32_t id) { _srcRequestorId = id; }
 
+    /** Setter for _partitionId */
+    void
+    setPartitionId(const uint64_t partitionId){ _partitionId = partitionId; }
```

Comment:
Space between ) and {

Reply:
Fixed this (and other instances of the same issue) in [commit 0bed2f7](https://github.com/gem5/gem5/pull/765/commits/0bed2f74c40bd95ae1529950eff0e751c4775572)